index.html

```diff
@@ -1,67 +1,218 @@
+                    <div class="yogalady"> <img src="image/yogalady.png" alt="Yogalady"></div>
+                    <div class="multi-post1">Multi-Post Stories</div>
+                    <div class="para-works1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+                    <div class="list1">
+                        <ul class="boxes1">
+                            <li class="box box1"><img src="image/css1.png" alt="CSS"></li>
+                            <li class="box box2"><img src="image/html1.png" alt="HTML"></li>
+                            <li class="box box3"><img src="image/bootstrap1.png" alt="Bootstrap"></a></li>
+                            <li class="box box4"><img src="image/ruby1.png" alt="Ruby"></li>
+                        </ul>
+                    </div>
+                    <div><button class="button1">See Project</button></div>
+                </div>
+    
+                <div class="works2">
+                    <div class="image-back">
+                        <div class="professional">Profesional Art Printing Data</div>
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works3">
+                    <div class="image-back">
+                        <div class="professional">Profesional Art Printing Data</div>
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
```

Comment:
- [ ] Your first project is looking great, however, I noticed that the **text** for the rest of your projects is not positioned correctly. As per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), it's important to the as possible to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=ZJcVC8XUqegdTgGs-0).
Would you be able to adjust the position of the text for the rest of your projects, following the same format as the first one that is looking great? This will help ensure that all of your cards are visually appealing and consistent.

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/235326292-78ad609b-ab1d-4238-80b2-4b8584d62045.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/235326310-d2ac7a84-56a5-4bb6-ae3f-0184faf2260c.png)